rmp-serialize/Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
 name = "rmp-serialize"
-version = "0.8.0"
+version = "0.8.0-rip"
```

Comment:
If we release a new version for marking the deprecation, won't the `-rip` will make it a pre-release? I think it will stop the new README from showing up at https://crates.io/crates/rmp-serialize.

Reply:
You're right.